CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# How to Contribute
+1. The operator-builder team welcomes contributions from the community.  Before you start working with operator-builder, please read our [Developer Certificate of Origin](https://cla.vmware.com/dco).  All contributions to this repository must be signed as described on that page.  Your signature certifies that you wrote the patch or have the right to pass it on as an open-source patch.
+1. Open an issue with what you intend to fix or would like to add into the project
```

Comment:
Small nit: Add a period (`.`) to the end of each list item for consistency.  Alternatively, remove all periods at the end of each list item for consistency.